game/graphics/renderer.cpp

```diff
@@ -168,6 +169,10 @@ void Renderer::resetSwapchain() {
   tonemapping.pso     = &Shaders::inst().tonemapping;
   tonemapping.uboTone = device.descriptors(*tonemapping.pso);
 
+  uint32_t availableFxaaPreset = std::clamp(Gothic::options().fxaaPreset, 0u, static_cast<uint32_t>(Shaders::FxaaPreset::LAST) - 1);
```

Comment:
`options().fxaaPreset` should be sanitized when parsing command-line, not at runtime.